src/app/market/listings/listings.component.ts

```diff
@@ -185,9 +185,11 @@ export class ListingsComponent implements OnInit, OnDestroy {
   }
 
   loadNextPage() {
-    let nextPage = this.getLastPageCurrentlyLoaded(); nextPage++;
```

Comment:
@vikas-cis What does this change relate to? ie: What is this change fixing? It doesn't seem to be related to any of the other changes.

Reply:
@zaSmilingIdiot yeah it was just kind of weird thing that happen when open your dev tool and when there are no listing then you have scroller on page and at some moment you scroll the page and you see the weird syntax error which has been fixed via the above fix.